src/main/java/Duke.java

```diff
@@ -1,10 +1,357 @@
+    static final String DEADLINE_ICON = "D";
+
+    //Data
+    static ArrayList<Task> list = new ArrayList<>();
+
+    private static void printLogo() {
+        String logoMessage = "Hello from\n" +
+                " ____        _        \n" +
+                "|  _ \\ _   _| | _____ \n" +
+                "| | | | | | | |/ / _ \\\n" +
+                "| |_| | |_| |   <  __/\n" +
+                "|____/ \\__,_|_|\\_\\___|";
+        System.out.println(logoMessage);
+    }
+
+    private static void greetUser() {
+         String greeting = DIVIDER + System.lineSeparator() + "Hello! I'm Jarvis!"
+                 + System.lineSeparator() + "What can I do for you?"
+                 + System.lineSeparator() + DIVIDER + System.lineSeparator();
+         System.out.println(greeting);
+    }
+
+    private static void sayByeToUser() {
+        String bye = DIVIDER + System.lineSeparator() + "Bye. Hope to see you again soon!"
+                + System.lineSeparator() + DIVIDER + System.lineSeparator();
+        System.out.println(bye);
+    }
+
+    private static void printList() {
+        String listMessage = DIVIDER + System.lineSeparator() + "Here are the tasks in your list:";
+        System.out.println(listMessage);
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i+1) + "." + list.get(i).toString());
+        }
+        System.out.println(DIVIDER);
+    }
+
+    private static void markTask(int taskNum) {
+        list.get(taskNum-1).markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(DIVIDER + System.lineSeparator() + list.get(taskNum-1).toString()
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void unmarkTask(int taskNum) {
+        list.get(taskNum-1).markAsNotDone();
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println(DIVIDER + System.lineSeparator() + list.get(taskNum-1).toString()
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void acknowledgementMessage() {
+        String acknowledgement = DIVIDER + System.lineSeparator()
+                + "Got it. I've added this task: " + System.lineSeparator()
+                + list.get(list.size()-1).toString();
+        System.out.println(acknowledgement);
+        System.out.println("Now you have " + (list.size()) + " task(s) in the list."
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void deleteTask(int taskNum) {
+        String acknowledge = DIVIDER + System.lineSeparator() + "Noted. I've removed this task: "
+                + System.lineSeparator() + list.get(taskNum-1).toString() + System.lineSeparator()
+                + "Now you have " + (list.size()-1) + " tasks in the list." + System.lineSeparator() + DIVIDER;
+        System.out.println(acknowledge);
+        list.remove(taskNum-1);
+    }
```

Comment:
Methods such as these should be packaged under a different class .  
e.g
1) print message methods could be packaged under a message handler class. 
2) task methods could be packaged under a separate task handler class too.